internal/bcdb/ledger_query_procesor_test.go

```diff
@@ -330,6 +354,84 @@ func constructProvenanceEntriesForDataTx(tx *types.DataTx, version *types.Versio
 	return txpData
 }
 
+func applyBlockOnStateTrie(trie *mptrie.MPTrie, worldStateUpdates []*worldstate.DBUpdates) error {
```

Comment:
As this is an exact duplicate of the same function in the `committer.go`, we can extract the common part of an exported function and then call it here and in the committer package